java/lib/src/main/java/com/svix/exceptions/ApiException.java

```diff
@@ -0,0 +1,44 @@
+package com.svix.exceptions;
+
+import java.util.Map;
+import java.util.List;
+
+public class ApiException extends Exception {
+    private int code = 0;
+    private Map<String, List<String>> responseHeaders = null;
+    private String responseBody = null;
+
+    public ApiException(final String message, final Throwable throwable, final int code, final Map<String, List<String>> responseHeaders, final String responseBody) {
```

Comment:
Maybe we should accept com.svix.internal.ApiException instead?
Then we can change below to call getCode of that member

Another option is to extend com.svix.internal.ApiException and not create the public method

Reply:
I originally had it accept a `com.svix.internal.ApiException` but IMO it felt like it broke the point of having `com.svix.internal` if its publicly exposed in a constructor in `com.svix.exceptions`

Extending sounds great though, done!

Reply:
Got it, though now we expose it in the Utils class, not sure if that's better. Should me move Utils to an internal package?

Reply:
Isn't `Utils` only accessible form within `com.svix`?

Reply:
Right, sorry. 
Anyway, it's fine to keep as is. You can change the constructor to be private package if you prefer (and accept com.svix.internal.ApiException).

Reply:
ah right 🤦. great idea!

Reply:
We can move it to `com.svix`, though dunno if that's better. Up to you, both sounds ok to me